make/osx.mk

```diff
@@ -45,6 +45,15 @@ USE_CUDA = 0
 # USE_CUDA_PATH = /usr/local/cuda
 USE_CUDA_PATH = NONE
 
+# CUDA architecture setting: going with all of them.
+# For CUDA < 6.0, comment the *_50 lines for compatibility.
+CUDA_ARCH := -gencode arch=compute_20,code=sm_20 \
```

Comment:
No. I missed that. But turns out Jenkins's cuda doesn't support 6.0.
04:01:28 nvcc fatal   : Unsupported gpu architecture 'compute_60'. I have to revert latest change.